git-hooks/post-checkout

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 
 echo "rebuilding extension"
-(npm install && npm run build:all) || echo "fail to rebuild javascript"
+(npm install && npm run build) || echo "fail to rebuild javascript"
```

Comment:
Sorry, forgot these existed.  They should be calling `build:main` to run the webpack build as well.